README.md

```diff
@@ -1,2 +1,82 @@
-# subspace-chain-alerts
-Important event alerts for subspace blockchains
+## Subspace Chain Alerts (PoC)
+
+Important event alerts for Subspace blockchains.
+
+### Status: Proof of Concept
+
+- This repository is in a PoC state. Interfaces, configuration, and behavior may change without notice.
+- Hardcoded settings are used for speed of iteration (Slack channel, WebSocket URL, thresholds, etc.).
+
+### What it does currently
+
+- Connects to a local Subspace node over WebSocket (`ws://127.0.0.1:9944`).
+- Keeps runtime metadata up to date via a background updater.
+- Posts Slack messages to a test channel when:
+  - The process launches and connects to the node (first observed block).
+  - Any `Sudo::*` extrinsic is included in a best block.
+  - Any `Balances::force*` extrinsic is included.
+  - Any `Balances::*` extrinsic whose transfer value exceeds a high threshold.
+- Slack messages include block height, time, hash, and the chain genesis hash, plus a truncated view of extrinsic bytes and decoded fields for quick triage.
+- Rate-limited Slack client with generous retries to avoid message loss on transient failures.
+
+### Current defaults and hardcoded values
+
+- Slack OAuth secret file path: `slack-secret` (file-based secret, not env vars).
+- Slack Workspace (team) ID: `T03LJ85UR5G` (Autonomys workspace).
+- Slack channel: `chain-alerts-test` (resolved to channel ID at startup).
+- Bot name and icon: "Teor's Chain Alerts Tester" and `:flag-au:`.
+- Node WebSocket: `ws://127.0.0.1:9944`.
+- Amount unit: `AI3` (1 AI3 = 10^18 planck-like units).
+- Alert threshold for large balance changes: `1_000_000 AI3`.
+- Slack API retries: 30; request throttle: 2s; channel list page limit: 1000.
+
+### Security notes
+
+- The Slack OAuth token is loaded from a file and must be readable only by the current user.
+  - On Unix, the process enforces `0400` or `0600` permissions. Other modes cause a panic at startup.
+- The token is wrapped and zeroized on drop to reduce in-memory exposure.
+- Do not commit the token; keep it out of version control and CI logs.
+
+### Limitations (PoC)
+
+- Hardcoded Slack channel, workspace ID, bot name/icon, node URL, and thresholds.
+- Best blocks subscription only; no finalized block selection logic yet.
```

Comment:
This is incorrect or misleading, we deliberately don't wait for blocks to finalise before alerting.